src/Controls/src/Core/Element/Element.cs

```diff
@@ -627,13 +627,21 @@ protected virtual void OnParentSet()
 			(this as IPropertyPropagationController)?.PropagatePropertyChanged(null);
 		}
 
+		Dictionary<string,bool> _properties = new Dictionary<string, bool>();
+		private protected override void OnPropertyChangedBindableProperty(BindableProperty property, object original, object value)
+		{
+			_properties[property.PropertyName] = true;
+			base.OnPropertyChangedBindableProperty(property, original, value);
+			_properties.Remove(property.PropertyName);
+		}
+
 		/// <summary>Method that is called when a bound property is changed.</summary>
 		/// <param name="propertyName">The name of the bound property that changed.</param>
 		protected override void OnPropertyChanged([CallerMemberName] string propertyName = null)
 		{
-			base.OnPropertyChanged(propertyName);
+			bool fireHandlerUpdate = !_properties.ContainsKey(propertyName);
 
-			UpdateHandlerValue(propertyName);
+			base.OnPropertyChanged(propertyName);
```

Comment:
I am not 100% sure what this is supposed to be doing? Maybe a comment as to why this is being delayed